test/basic-querying.test.js

```diff
@@ -469,6 +469,58 @@ describe('basic-querying', function() {
       });
     });
 
+    it('should support "like" that is satisfied', function(done) {
+      User.find({where: {name: {'like': 'John'}}}, function(err, users) {
+        should.not.exist(err);
```

Comment:
Thanks, this is fine. All those other parts of the file need to change to this style anyways (if you would like to submit a PR for that, I can help approve that one too ;)).